src/generate_test_vectors.py

```diff
@@ -165,33 +165,33 @@ def ct_max_len(self):
 
 
 class ECDHP256KEM(ECDHKEM):
-  id = "ECDH-P256"
   oid = univ.ObjectIdentifier((1,2,840,10045))
```

Comment:
From a quick scan, `oid` is never used and should be removed from all classes to avoid confusion.